src/Generating/StructGen.cpp

```diff
@@ -204,87 +221,88 @@ int cStructGenTrees::GetNumTrees(
 	{
 		switch (a_Biome)
 		{
-			case biOcean:                return 2;
-			case biPlains:               return 1;
-			case biDesert:               return 0;
-			case biExtremeHills:         return 3;
-			case biForest:               return 30;
-			case biTaiga:                return 30;
-			case biSwampland:            return 8;
-			case biRiver:                return 0;
-			case biNether:               return 0;
-			case biEnd:                  return 0;
-			case biFrozenOcean:          return 0;
-			case biFrozenRiver:          return 0;
-			case biIcePlains:            return 1;
-			case biIceMountains:         return 1;
-			case biMushroomIsland:       return 3;
-			case biMushroomShore:        return 3;
-			case biBeach:                return 0;
-			case biDesertHills:          return 0;
-			case biForestHills:          return 20;
-			case biTaigaHills:           return 20;
-			case biExtremeHillsEdge:     return 5;
-			case biJungle:               return 120;
-			case biJungleHills:          return 90;
-			case biJungleEdge:           return 90;
-			case biDeepOcean:            return 0;
-			case biStoneBeach:           return 0;
-			case biColdBeach:            return 0;
-			case biBirchForest:          return 30;
-			case biBirchForestHills:     return 20;
-			case biRoofedForest:         return 50;
-			case biColdTaiga:            return 20;
-			case biColdTaigaHills:       return 15;
-			case biMegaTaiga:            return 30;
-			case biMegaTaigaHills:       return 25;
-			case biExtremeHillsPlus:     return 3;
-			case biSavanna:              return 8;
-			case biSavannaPlateau:       return 12;
-			case biMesa:                 return 2;
-			case biMesaPlateauF:         return 8;
-			case biMesaPlateau:          return 8;
+			case biOcean:                return 2.0;
+			case biPlains:               return 0.03125;
+			case biDesert:               return .0;
```

Comment:
Please don't use the horrible "nothing point something" format. And anyway, what's wrong with a plain zero?

Reply:
0 doesn't work because of auto
.0 and 0.0 works, I have change it.